digital_image_processing/change_contrast.py

```diff
@@ -28,6 +28,8 @@ def contrast(c: int) -> float:
 
 
 if __name__ == "__main__":
+    import doctest
+    doctest.testmod()

```

Comment:
It is not necessary to run the tests every time that the script is run.

Reply:
Removed 👍